package/MDAnalysis/analysis/density.py

```diff
@@ -763,7 +783,8 @@ def convert_length(self, unit='Angstrom'):
         """
         if unit == self.units['length']:
             return
-        cvnfact = units.get_conversion_factor('length', self.units['length'], unit)
```

Comment:
I'm not really following what happened here. Why do you think these changes are causing issues?

Reply:
There are a few changes which were made by autoPEP8 and not a part of fixing the error. The only failing check of CodeCov is because these changes are not covered by tests..

Reply:
It's up to you, codecov is complaining because it's gone from one line of untested code to two. The best solution here would be to go ahead and add a test for this, but otherwise I think we can live with just a style change.

Reply:
Sounds good! Thanks @IAlibay :)